scripts/wait_for_progress.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+#
+# usage:
+#  wait_for_progress.py node.log
+#
+# wathches log file for round advancement.
+
+import json
+import queue
+import subprocess
+import sys
+import threading
+import time
+
+def tailf(fname):
+    pr = subprocess.Popen(['tail', '-F', '-n', '1', fname], stdout=subprocess.PIPE)
+    for line in pr.stdout:
+        yield line
+    return
+
+_timeout_ob = {}
+
+def timeout_linesource(linesource, timeout=60):
```

Comment:
`goal node wait`

Reply:
ayup. deleted this whole script and used that instead.